backend/tracim_backend/__init__.py

```diff
@@ -37,6 +37,7 @@
 from tracim_backend.views.frontend import FrontendController
 from tracim_backend.views.errors import ErrorSchema
 from tracim_backend.exceptions import NotAuthenticated
+from tracim_backend.exceptions import ContentInNotEditableState
```

Comment:
ContentInNotEditableState is the correct name ? Or it should be ContentI**s**NotEditableState ?

Reply:
I don't think so, content is not a state, content has as state.